src/commands/install.yml

```
@@ -61,7 +61,9 @@ steps:
   - when:
       condition: << parameters.install-browsers >>
       steps:
-        - browser-tools/install-browser-tools
+        - browser-tools/install_browser_tools:
+            install_chrome_for_testing: true
+            install_chrome_for_testing_driver: true
```

```suggestion
            install_edge: true
```

The driver isn't needed, however Edge is missing